quodlibet/quodlibet/images/hicolor/Makefile

```diff
@@ -0,0 +1,82 @@
+QL_SOURCE=scalable/apps/quodlibet.svg
+QL_DEST=quodlibet.png
+
+QL_SYM_SOURCE=scalable/apps/quodlibet-symbolic.svg
```

Comment:
They can be converted, but gtk+ is not able to use them. Just remove the targets/variables.